### PR TITLE
Add workstation team as code-owner

### PR DESCRIPTION
**Pull request by vkarve-chef**
Signed-off-by: Vikram Karve <vikram.karve@progress.com>

## Description
this repo will be maintained by the workstation team going forward so need ability to approve and merge PRs.

## Related PR 
https://github.com/chef/chef-oss-practices/pull/354

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Order is important. The last matching pattern has the most precedence.
 
-*            @chef/chef-infra-reviewers @chef/chef-infra-approvers @chef/chef-infra-owners
+*            @chef/chef-infra-reviewers @chef/chef-infra-approvers @chef/chef-infra-owners @chef/chef-workstation-approvers
```
Comment (tas50) [inline .github/CODEOWNERS:3]: You want to add all 3 groups

Comment (tas50) [reply .github/CODEOWNERS:3]: Make sure to PR this repo into this list too https://github.com/chef/chef-oss-practices/blob/main/projects/chef-workstation.md

Comment (tas50) [reply .github/CODEOWNERS:3]: If Workstation team is going to take on this repo then Infra should probably get moved off as owners as well.

Comment (vkarve-chef) [reply .github/CODEOWNERS]: done, created https://github.com/chef/chef-oss-practices/pull/354 too

**Pull request merged**